docs/api/app.md

```diff
@@ -720,9 +722,9 @@ In that case, the directory should be created with `fs.mkdirSync` or similar.
 
 You can only override paths of a `name` defined in `app.getPath`.
 
-By default, web pages' cookies and caches will be stored under the `userData`
+By default, web pages' cookies and caches will be stored under the `userCache`
 directory. If you want to change this location, you have to override the
-`userData` path before the `ready` event of the `app` module is emitted.
+`userCache` path before the `ready` event of the `app` module is emitted.
```

Comment:
@jkleinsc this arguably makes this `semver/major` in the case of people who were already customizing their cookie location with code that overrides `userData`?

Reply:
If people are customizing 'userData' the behavior remains the same. 'userCache' is a new way to move the cache out of its default location in 'userData' (in my case on Windows, move it out of the roaming profile of the user)